db/migrate/20171110120821_create_decidim_census_censuses.decidim_censuses.rb

```diff
@@ -0,0 +1,12 @@
+# This migration comes from decidim_censuses (originally 20171110102910)
+class CreateDecidimCensusCensuses < ActiveRecord::Migration[5.1]
+
+  def change
+    create_table :decidim_censuses_censuses do |t|
+      t.string :id_document
+      t.string :birthdate
```

Comment:
I have a question. Why we save the birthdate as a string? I see more advantages of using a date object.

Reply:
Agree. It departures from the standard, and thats not a good thing. Also it's not clear if it's really a date, and it's not documented the format. So, rolling back to a date field!